Firestore/Protos/protos/firestore/local/target.proto

```diff
@@ -22,6 +36,7 @@ message Target {
   // The last snapshot version received from the Watch Service for this target.
   //
   // This is the same value as TargetChange.read_time
+  // https://github.com/googleapis/googleapis/blob/master/google/firestore/v1beta1/firestore.proto#L734
```

Comment:
I don't much care, but this comment could go stale and end up pointing to the wrong thing if firestore.proto is changed in the future.

Solutions:
1) Nothing. If it changes, the reader will figure it out.
2) Embed the commit hash in the url, i.e. https://github.com/googleapis/googleapis/blob/02111201c4752cdd0d7a5714ec6644f1d0297f51/google/firestore/v1beta1/firestore.proto#L734
3) Omit "L734" and let the reader ctrl+f.